table/multiget_context.h

```diff
@@ -22,6 +24,7 @@ struct KeyContext {
   const Slice* key;
   LookupKey* lkey;
   Slice ukey;
+  Slice ukey_without_ts;
```

Comment:
Assuming this is kept, it's probably good to update existing calls to StripTimestampFromUserKey in memtable.cc

Reply:
Sorry, those might not be for multiget, but I'm pretty sure memtable Bloom multiget is also broken (up to you whether to fix in this PR).

Reply:
thanks @pdillinger, updated the calls to memtable bloomfilter.